core/types/Interfaces.ts

```diff
@@ -284,7 +284,7 @@ export interface MenuItem {
     className?: string;
 
     /** Executed when the user clicks the menu item. */
-    actionFn?: () => void;
+    actionFn?: (e: MouseEvent) => void;
```

Comment:
Is this a `MouseEvent` on mobile?  Wondering if it's actually a `TouchEvent` there and if we should spec either `UIEvent` or `MouseEvent | TouchEvent` in this core interface?